apps/server/src/routes/book/index.ts

```diff
@@ -0,0 +1,67 @@
+import express, {Express} from 'express'
+
+import { prisma } from '../../lib/prismaClient'
+
+export function bookRoute(app: Express): void {
+    const router = express.Router()
+    
+    
+    app.use('/api/book', router)
+
+    router.get('/', async (req, res) => {
+      try {
+        const products = await prisma.product.findMany();
+        res.json(products);
+      } catch (error) {
+        res.status(500).json({ error: 'Failed to fetch products' });
+      }
+    });
+    router.post('/', async (req, res) => {
+        const {name,author,image,price,variation,collection} = req.body;
+        try {
+          const newBook = await prisma.product.create({
+            data: {
+              name,
+              author,
+              image,
+              price,
+              variation: { connect: { id: 1 } },
+              collection: { connect: { id: 1 } }
+            },
+          });
+          res.status(201).json(newBook);
+        } catch (error) {
+          res.status(500).json({ error: 'Failed to create product' });
+        }
```

Comment:
HINT: instead of performing all the logic in the route, this can be moved to the bookServices.